zhaquirks/tuya/ts011f_plug.py

```diff
@@ -36,6 +37,13 @@
 from zhaquirks.tuya.mcu import EnchantedDevice
 
 
+class TuyaZBOnOffChildLockAttributeCluster(CustomCluster, OnOff):
```

Comment:
Maybe `TuyaZBOnOffChildLockCluster`?

Reply:
Have renamed to `TuyaZBOnOffVariantCluster` since it turns out that the `power_state` attribute actually is supported. See comment [below](https://github.com/zigpy/zha-device-handlers/pull/2085#issuecomment-1382569133) for further detail.